model/config.go

```diff
@@ -752,6 +753,19 @@ func (o *Config) SetDefaults() {
 		*o.EmailSettings.EmailBatchingInterval = EMAIL_BATCHING_INTERVAL
 	}
 
+	if o.EmailSettings.EnableSMTPAuth == nil {
```

Comment:
It looks fine to me. It's setting a default, but it's doing it intelligently based on the other setting.